### ISSUE TITLE
Cluckshroom

### ISSUE THREAD

**Issue by cech12** [cech12/ExtendedMushrooms#40]
Add a cluckshroom to the mod. 

It is a chicken like mob that leave a trail of mushrooms when it walks in the dark. They seek dark areas.
If a cluckshroom is under direct sunlight, it runs around wildly until it is out of the sun.

* https://minecraft.gamepedia.com/Minecraft_Earth:Cluckshroom
* see also the Earth Mobs mod https://github.com/baguchan/EarthMobs
https://www.curseforge.com/minecraft/mc-mods/earth-mobs 
* https://www.curseforge.com/minecraft/mc-mods/minecraft-earth-mod

**Comment by Aceplante** [comment]
I agree a mushroom chicken would fit perfectly in with the theming of this mod, perhaps plants mushrooms occasionally instead of laying eggs. I personally would not want it to "leave a trail" constantly but the preferring darkness mechanic seems cool. Perhaps only "lays" the mushroom in dark areas?
